src/internal_events/kubernetes_logs.rs

```diff
@@ -9,6 +9,7 @@ pub struct KubernetesLogsEventsReceived<'a> {
     pub file: &'a str,
     pub byte_size: usize,
     pub pod_name: Option<&'a str>,
+    pub pod_namespace: Option<&'a str>,
```

Comment:
My suggestion would be to make this:

```rust
    pub pod_metadata: Option<&'a PodMetadata>,

// ...

struct PodMetadata <'a> {
  namespace: &'a str
  name: &'a str
}
```

That way if it is set at all, you can fetch the needed data.

Alternatively you could share the `LogFileInfo` struct and have the above be:

```
pub pod_metadata: Option<&'a LogFileInfo>,
```